### PR TITLE
Marked JetBrains "Coming soon" → experimental

### PR DESCRIPTION
**Pull request by vdavid**
Cody for JetBrains is actually out. Added it to the Cody landing.

Before:

CleanShot 2023-07-20 at 12 26 27@2x

After:

CleanShot 2023-07-20 at 12 23 54@2x

## Test plan

Waiting for reviews and CI

### CODE REVIEW

**Review by jdorfman — APPROVED** [review]
🔥

File: client/web/src/storm/pages/SearchPage/TryCodySignUpCtaSection.tsx [hunk 1]
```diff
@@ -101,10 +101,13 @@ export const TryCodySignUpCtaSection: React.FC<TelemetryProps & { className?: st
                         VS Code extension
                     </Text>
                     <Text as="li" size="small" className="mb-2">
-                        the Sourcegraph web application{' '}
+                        the Sourcegraph web application
+                    </Text>
+                    <Text as="li" size="small" className="mb-2">
+                        our JetBrains IDE plugin <ProductStatusBadge status="experimental" />
```
Comment (toolmantim) [inline client/web/src/storm/pages/SearchPage/TryCodySignUpCtaSection.tsx:107]: The default styling of `<ProductStatusBadge status="experimental" />` is quite intense hey! A simple outline style here would suffice I think 🤔

Comment (toolmantim) [reply client/web/src/storm/pages/SearchPage/TryCodySignUpCtaSection.tsx:107]: (i.e. we probably don't want it to be the most prominent thing on the page)

Comment (vdavid) [reply client/web/src/storm/pages/SearchPage/TryCodySignUpCtaSection.tsx:107]: @toolmantim Better?

CleanShot 2023-07-21 at 13 51 27@2x

Comment (toolmantim) [reply client/web/src/storm/pages/SearchPage/TryCodySignUpCtaSection.tsx]: Much better! ty ty

**Review by toolmantim — APPROVED** [review]

**Pull request merged**